docs/guides/lint-asyncapi.md

```diff
@@ -53,8 +53,8 @@ some of the built-in rules. The currently-supported rules are:
 
 - `info-contact`: the `Info` section must contain a valid `Contact` field.
 - `operation-operationId`: every operation must have a valid `operationId`.
-- `channels-kebab-case`: channel names should be `kebab-case` (lowercase with hyphens).
-- `no-channel-trailing-slash`: channel names must not have trailing slashes in their names.
+- `channels-kebab-case`: channel address should be `kebab-case` (lowercase with hyphens).
```

Comment:
I was confused when I didn't find `name` property in channel, I guess it should be `address`.
Please correct me if I am wrong, @lornajane | @tatomyr .

Reply:
Seems like we could also have an "AsyncAPI" group?

Reply:
Reformatted the overall structure and these individual rules pages.